DirectProgramming/DPC++FPGA/Tutorials/Features/experimental/register_map_and_streaming_interfaces/README.md

```diff
@@ -0,0 +1,324 @@
+
+
+# Register Map and streaming interfaces
+This FPGA tutorial demonstrates how to specify the kernel control interfaces and kernel argument interfaces. The kernel control interfaces can be implemented as register map interfaces or streaming interfaces. Similarly, the kernel argument interfaces can be implemented as register map interfaces or streaming interfaces as well, independent of which interface that the kernel control is using. The register map interface is referring to an interface registered in the kernel memory map, where the streaming interface is referring to an interface that signals are implemented in simple conduits.
```

Comment:
Did we consciously re-name the start/busy/stall/done interface from **Invocation Interface** (in HLS) to **Control Inteface** (In IP Authoring)?

This may seem pedantic but we really need to make sure that these things all have consistent names or customers will have no idea what we are talking about.

Reply:
No there was no conscious renaming - I think these are the terms we have been using internally. Let us stay consistent with the HLS terms then? Paul, not also the use of conduits, streaming and register map in the tutorial - register-map was consciously chosen new name for 'agent' we hope to make that clear in the docs/tutorial.

Reply:
I have updated all references to `Control Interfaces` to `Invocation Interfaces` in this sample.